backend/src/main/repositories/EmployeeHoursInputTypeRepository.ts

```diff
@@ -0,0 +1,98 @@
+import debug from 'debug';
+import { injectable } from 'tsyringe';
+import { CRUD } from './CRUDInterface';
+const log: debug.IDebugger = debug('app:FeedbackRepository');
+import {
+  EmployeeHoursInputTypeCreationDTO,
+  EmployeeHoursInputTypeUpdateDTO,
+} from '../dto/LogHours/EmployeeHoursInputTypeDTOs';
+import { EmployeeHoursInputType } from '../models/EmployeeHoursInputType';
+
+@injectable()
+export default class EmployeeHoursInputTypeRepository implements CRUD {
+  constructor() {
+    log('Created new instance of EmployeeHoursInputTypeRepository');
+  }
+
+  public create = async (
+    employeeHoursInputTypeCreationInfo: EmployeeHoursInputTypeCreationDTO
+  ): Promise<EmployeeHoursInputType> => {
+    try {
+      const createdEmployeeHoursInputType = EmployeeHoursInputType.build(employeeHoursInputTypeCreationInfo);
+      createdEmployeeHoursInputType.save();
```

Comment:
add await here because without it, we wouldn't be able to catch an error on time due to async.